src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,82 @@
+// Blank table
+>> Distinct(If(1<0,[1,2]), true)
+Blank()
+
+// Empty table
+>> Distinct(Filter([1,2,3],Value>10), true)
+Table()
+
+// Constant expressions
+>> Distinct(Table({a:1,b:2},{a:3,b:4},{a:5,b:6}), true)
+Table({Result:true})
+
+>> Distinct(Table({a:1,b:2},{a:3,b:4},{a:5,b:6}), false)
+Table({Result:false})
+
+// Distinct values for a column
+>> Distinct(Table({a:1,b:2},{a:3,b:4},{a:5,b:6}), a)
+Table({Result:1},{Result:3},{Result:5})
+
+// Distinct values for an expression
+>> Distinct(Table({a:1,b:2},{a:3,b:4},{a:5,b:6}), a * 2)
+Table({Result:2},{Result:6},{Result:10})
+
+>> Distinct(Table({a:1,b:2},{a:3,b:4},{a:5,b:6}), Trunc(a / 2))
+Table({Result:0},{Result:1},{Result:2})
+
+// Distinct values for an expression that generates same result
+>> Distinct([-2,-1,0,1,0],Value^2)
+Table({Result:4},{Result:1},{Result:0})
+
+// Distinct values with blanks
+>> Distinct(Table({FirstName:"Harry"},{FirstName:Blank()},{FirstName:"Sally"},{FirstName:"Harry"},{FirstName:Blank()},{FirstName:"Alice"},{FirstName:"Sally"}), FirstName)
```

Comment:
>"Harry" [](http://example.com/codeflow?start=29&length=7)

Can we add a test for case insensitivity. 

"Harry", "HARRY",  ... same of different?

Reply:
added